configs/setups/awiesm3/awiesm3.yaml

```diff
@@ -0,0 +1,783 @@
+                "v3.4":
+                        version: "2.6.8"
+                        namelist_dir: "${esm_namelist_dir}/fesom2/2.6/awicm3/v3.4/"
+                        add_namelist_changes:
+                                namelist.config:
+                                        ale_def:
+                                                which_ALE: 'zstar'
+                                namelist.oce:
+                                        oce_dyn:
+                                                SPP: true
+                                                k_gm_max: 3000
+                                                k_gm_rampmax: 40.0
+                                                k_gm_rampmin: 30
+                                                scaling_ferreira: true
+                                                scaling_fesom14: true
+                                        oce_tra:
+                                                surf_relax_s: "${surf_relax_s}"
+
+                                namelist.tra:
+                                        tracer_phys:
+                                                surf_relax_s: "${surf_relax_s}"
+                                                use_momix: false
+                                                kv0_const: false
+                                namelist.ice:
+                                        ice_therm:
+                                                albsn: 0.81
+                                                albsnm: 0.77
+                                                albi: 0.6
+                                                albim: 0.43
+
+
+        # Remove ICMCL_INIT forcing, which is only required for standalone simulations (form v3.2 onwards contains seasonal leaf area index, veg albedo etc.)
+        remove_forcing_files:
+                - ICMCL_INIT
+        remove_namelist_changes.namelist.config.inout: # Not present in the fesom-bounded namelist but in the fesom-2.0.yaml
+                - restart_length
+                - restart_length_unit
+        remove_namelist_changes.namelist.config.paths:
+                - ForcingDataPath
+                - OpbndPath
+                - TideForcingPath
+        # Remove all namelist changes for namelist.forcing defined in fesom-2.0.yaml
+        remove_namelist_changes:
+                - namelist.forcing
+
+
+        coupling_freq_in_steps: $((${oasis3mct.coupling_time_step} / ${fesom.time_step}))
+
+        remove_config_files:
+                - diag
+
+        atm_oce_fields: [ist_feom, sia_feom]
+
+        coupling_fields:
+                "[[atm_oce_fields-->FIELD]]":
+                        grid: feom
+
+
+# It is possible for the esm_tools to modify the fesom io namelist, in case you don't want to make your own copy and edit that.
+# By default this option is turned off, and thus the follwing two lines are commented out.
+# further_reading:
+#       - fesom/fesom.io.yaml
+
+lpj_guess:
```

Comment:
How much of this belongs to the setup and how much actually to the component?

I have the impression that a lot can be moved to the lpj_guess.yaml

Reply:
This is true, but not of great concern until LPJG is used for other setups. I think this day is far away.